### ISSUE TITLE
Not as effective with PrusaSlicer's new "Pressure Equalization" setting

### ISSUE THREAD

**Issue by jhstatewide** [FormerLurker/ArcWelderPlugin#249]
PrusaSlicer introduced a new beta feature: https://help.prusa3d.com/article/pressure-equalizer-_331504

When this is used in combination with ArcWelder, the total amount of arcs created is much less. I would theorize that is because ArcWelder can only weld segments that have identical speed / acceleration. 

If the above is true, is there any hope for ArcWelding these files, or are the two methods entirely incompatible?

**Comment by zapl** [comment]
That's correct. Segments can only be welded together if the settings don't change. There needs to be a new GCODE command every time the printer needs to do something different.
But I would rather use firmware pressure advance (klipper) / linear advance (marlin) which do the same thing with the advantage that the firmware has much more real time control over the pressure than the slicer.
